src/pages/Explorer/Components/EmptyState.svelte

```diff
@@ -0,0 +1,20 @@
+<script>
+    import EmptyCreations from './EmptyCreations.svelte'
+    import { MenuItem } from '../const'
+
+    export let activeMenu;
+</script>
+
+<div class="empty row hv-center fluid mrg--t">
+    {#if activeMenu == MenuItem.LIKES}
+        <EmptyCreations id="rocket" action="like" title="No liked" />
+    {:else if activeMenu == MenuItem.MY_CREATIONS}
+        <EmptyCreations id="browser" action="make" title="No" />
+    {/if}
+</div>
+
+<style>
+    .empty {
+        --margin: 120px;
+    }
```

Comment:
To improve readability, better to set the margin as the style rule (`mrg--t` is not needed)

```suggestion
    .empty {
        margin-top: 120px;
    }
```